mobile/app/(observer)/(app)/report-issue.tsx

```diff
@@ -575,7 +647,10 @@ const ReportIssue = () => {
         gap="$sm"
       >
         {/* this will reset form to defaultValues */}
-        <Button preset="chromeless" onPress={() => reset()}>
+        <Button
+          preset="chromeless"
+          onPress={() => reset({ incident_category: IncidentCategory.Other })}
```

Comment:
aici nu e nevoie sa ii dai din nou valorea pentru incident_category, reset-ul va reseta valorile la cele puse in default